packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/isTraceValueVisible..st

```diff
@@ -0,0 +1,5 @@
+private
+isTraceValueVisible: aTraceValue
+
+	 ^ ((context ignoredAnnotationsOf: example) includes: (aTraceValue belongingAnnotationId)) not
```

Comment:
i'm skeptical of saving `example -> set of annotation ids` in the dictionary in `context`.
it seems a bit too much work to always iterate through a set when adding/removing/searching. all calls to `context` in the `BPSwimlane` class always have their use case focused on adding a single annotation rather than adding *all* annotations of an example. what were your thoughts on deciding against `annotation id -> boolean if it is ignored` ?

Reply:
I thought it's better to tie it to a specific example as the show/hide button is right next to annotation values in a swimlane - so one wouldn't expect a global effect, in my opinion.
Also, if you have 2 examples open in the printbugger, you might want to ignore values of an annotation for example 1, but I want to see them for example 2.

Reply:
ok, fair